test/index.js

```diff
@@ -136,10 +136,6 @@ describe('svgString() should retain camel-casing', function () {
   it('should return svg', function () {
     assert.equal(d3n.svgString(), '<svg xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="true"><radialGradient offset="0%"></radialGradient></svg>')
   })
-
-  it('should return html (w/o casing)', function () {
-    assert.equal(d3n.html(), '<html><head></head><body><svg xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="true"><radialgradient offset="0%"></radialgradient></svg></body></html>')
-  })
```

Comment:
why remove?

Reply:
This was removed since jsdom no longer has this name casing bug, so this test is not needed. The test can't pass since it will respect the casing. The other test ensures the casing happens properly, which is also why I removed the function that was patching it up and making casing work, working around the jsdom bug.